app/pulp/app/models/progress.py

```diff
@@ -44,14 +44,14 @@ class ProgressReport(Model):
     SKIPPED = 'skipped'
     RUNNING = 'running'
     COMPLETED = 'completed'
-    ERRORED = 'errored'
+    FAILED = 'failed'
```

Comment:
We need to make sure this gets into a list of backwards-incompatible changes. Perhaps it's time to start at least a skeleton for 3.0 release notes with a list of changes. It doesn't have to be well-written; we just need to not forget about "little" changes like this that have a huge impact on people's integration with pulp.